docs/app/documentation/component-docs/form/form-docs.component.html

```diff
@@ -68,13 +66,21 @@ <h2>Radio buttons</h2>
 
 <separator></separator>
 <h2>Checkbox</h2>
-<description>With checkboxes, all options are visible and the user can make one or more selections. This component can be set
-    disabled
-    and also displayed in a row.
-    <br/><br/>
+<description
+    >With checkboxes, all options are visible and the user can make one or more selections. This component can be set
+    disabled and also displayed in a row. <br /><br />
     Checkboxes are keyboard navigable via the tab key and selected with the space bar.
 </description>
 <component-example [name]="'ex6'">
     <fd-form-checkbox-example></fd-form-checkbox-example>
 </component-example>
 <code-example [code]="checkboxFormHtml" [language]="'HTML'"></code-example>
+
+<separator></separator>
+<h2>Form Group Input</h2>
+<description>Form Group Input </description>
```

Comment:
Could you provide a more detailed explanation here?

Reply:
Ok done